function-controller/src/main/java/io/sk8s/function/controller/FunctionDeployer.java

```diff
@@ -123,29 +117,11 @@ private PodSpec buildPodSpec(XFunction function) {
 	}
 
 	private Container buildMainContainer(XFunction function) {
-		ContainerBuilder builder = new ContainerBuilder().withName("main")
-				.withImage(function.getSpec().getImage())
-				.withImagePullPolicy("IfNotPresent");
```

Comment:
so this is changing our default from `IfNotPresent` to `Always` (can be overridden by a user now though)... possibly this is a good thing since that's the normal default. WDYT @trisberg ?

Reply:
Actually, it's only `Always` if image is tagged `latest` or no tag. See https://kubernetes.io/docs/concepts/containers/images/#updating-images

We can change, but I believe using the k8s semantics is better (users can specify now, as they have access to the full range of `container` props)